tests/tasks/rmfile/rm_wildcard.cue

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+)
+
+dagger.#Plan & {
+	actions: {
+		pull: core.#Pull & {
+			source: "alpine:3.15.0@sha256:e7d88de73db3d3fd9b2d63aa7f447a10fd0220b7cbf39803c803f2af9ba256b3"
+		}
+
+		test: {
```

Comment:
nit: slightly simpler to just put each of the three tests in the same `cue` file as separate targets

Reply:
Yeah I was thinking the same, it would be cleaner.
To simplify test, I grouped them into key `test` so we can execute them all as one in https://github.com/dagger/dagger/pull/2422/commits/e39926bccdb5ab6328c32cfc67f34574111c1d50.

That's maybe something we can replicate on other tasks tests btw

Reply:
Richard is the one that initially suggested we do this and we mostly do it except in older tests. There are other valid cases for splitting. If you need to test for failures, for example, or different behaviors from the environment like exporting different environment variables from bats. You see this a lot in the Client API tests because there's a lot of interaction with the host environment. 

Other than that there's a convention of putting all tests inside a `test` action so you can just do `dagger do test`:
- https://github.com/dagger/dagger/issues/1736